composer.json

```diff
@@ -28,7 +37,11 @@
     "prefer-stable": true,
     "scripts": {
         "post-install-cmd": "sh ./scripts/composer/post-install.sh",
-        "docker-swagger-generate": "docker-compose run swagger generate -i http://service:3000/explorer/swagger.json -l php -o /var/usr/client -c /var/usr/swagger/config.json"
+        "docker-swagger-generate": "docker-compose run swagger generate -i http://service:3000/explorer/swagger.json -l php -o /var/usr/client -c /var/usr/swagger/config.json",
+        "lint": "vendor/bin/parallel-lint . --exclude vendor --exclude web/core --exclude web/sites/*/files",
```

Comment:
Linting catches syntax errors. PHPCS catches code style deviations.

This is primarily for the deployment pipeline where we could deploy something that does not follow our code style but we should not deploy something that does not compile.